lib/internal/options.js

```diff
@@ -43,7 +47,7 @@ function refreshOptions() {
 
 function getOptionValue(optionName) {
   const options = getCLIOptionsFromBinding();
-  if (optionName.startsWith('--no-')) {
+  if (StringPrototypeStartsWith(optionName, '--no-')) {
     const option = options.get('--' + optionName.slice(5));
```

Comment:
Why not StringPrototypeSlice here?

Reply:
You're right, I will update the code later today.